packages/picasso-forms/src/FieldWrapper/FieldWrapper.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+import Field, {
+  Props as FieldProps,
+  ValueType,
+  IFormComponentProps
+} from '../Field'
+import FieldLabel from '../FieldLabel'
+
+export type Props<TWrappedComponentProps, TInputValue> = Omit<
+  FieldProps<TWrappedComponentProps, TInputValue>,
+  'label'
+> & { label?: string }
+
+const FieldWrapper = <
+  TWrappedComponentProps extends IFormComponentProps,
+  TInputValue extends ValueType = TWrappedComponentProps['value']
+>(
+  props: Props<TWrappedComponentProps, TInputValue>
+) => {
+  const { label, name, titleCase, children, ...rest } = props
+
+  return (
+    <Field<IFormComponentProps, TInputValue>
+      {...rest}
+      name={name}
+      label={
+        label ? (
+          <FieldLabel
+            name={props.name}
+            required={props.required}
+            label={label}
+            titleCase={titleCase}
+          />
+        ) : null
+      }
+    >
+      {children}
+    </Field>
+  )
```

Comment:
Do we need to be open this much?

```suggestion
  return (
    <Field<IFormComponentProps, TInputValue>
      {...rest}
      label={
        label ? (
          <FieldLabel
            name={props.name}
            required={props.required}
            label={label}
            titleCase={titleCase}
          />
        ) : null
      }
    />
  )
```
IMHO: all we need is to get the label so it is clear this way. wdyt?

Reply:
@ozgurkececioglu I initially did the same, but it looks like I have issues with types again (generics) 🤔 so in this way I could make it work. I can try to fix this if we are ready to wait for this PR a bit more 🤔

Reply:
No need, it is so minor. We can take a look later 👍